pkg/controller/bootstrap/jws.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import (
+	"fmt"
+	"strings"
+
+	jose "github.com/square/go-jose"
+)
+
+func computeDetachedSig(content, tokenID, tokenSecret string) (string, error) {
```

Comment:
Well, a detached signature is a thing so simply saying `signWithJWS` is misleading. And golang errs on the side of brevity.  And this is in a file called `jws.go` so that is a pretty much implied.